packages/app/src/sandbox/eval/transpilers/postcss/loader.ts

```diff
@@ -86,7 +116,50 @@ export default async function (
     );
   }
 
+  let transpiledCode = result.css;
   const map = result.map && result.map.toJSON();
 
-  return { transpiledCode: result.css, sourceMap: map };
+  async function _loadModule(filepath: string): Promise<string> {
+    // We load straight from the unpkg link if it's a node_module
+    if (filepath.startsWith('/node_modules')) {
+      const packageName = getPackageName(filepath);
+      const packageRoot = `/node_modules/${packageName}`;
+      const pkgJsonModule = await loaderContext.resolveTranspiledModuleAsync(
```

Comment:
Is there a way to prevent actual transpilation in this case, like just read the file but still resolve/download it?